src/js/src/helper.cpp

```diff
@@ -43,27 +43,49 @@ ov::Any js_to_cpp<ov::Any>(const Napi::Env& env, const Napi::Value& value) {
         }
     } else if (value.IsBoolean()) {
         return ov::Any(static_cast<bool>(value.ToBoolean()));
-    } else {
-        OPENVINO_THROW("Cannot convert to ov::Any");
+    } else if (value.IsArray()) {
+        return ov::Any(js_to_cpp<std::vector<std::string>>(env, value));
+    } else if (value.IsObject()) {
+        if (value.ToString().Utf8Value() == "[object Set]") {
```

Comment:
What js type is needed here? Maybe [this](https://github.com/openvinotoolkit/openvino/blob/5561c207aba860abeca4b383a7976f61433d8e1f/src/bindings/js/node/src/helper.cpp#L539) approach will be better?

Reply:
We check the type of Set here.
I tried to call `inferenceOf` for Set from global environment, but unfortunately that didn't work. The only reason I can see for the failure to check is that the object was created in a different global environment.
I don't really like the current approach, but for now  it can be used as a workaround.